chain/client/src/metrics.rs

```diff
@@ -152,3 +152,10 @@ pub static CHUNK_SKIPPED_TOTAL: Lazy<IntCounterVec> = Lazy::new(|| {
     )
     .unwrap()
 });
+pub static PARTIAL_ENCODED_CHUNK_RESPONSE_DELAY: Lazy<Histogram> = Lazy::new(|| {
+    try_create_histogram(
+        "partial_encoded_chunk_response_delay",
+        "Delay between when a partial encoded chunk response is sent from PeerActor and when it is received by ClientActor",
+    )
```

Comment:
This doesn't tell us whether the delay is between peer actor and peer manager actor or between peer manager actor and client actor

Reply:
The response is sent directly from PeerActor to ClientActor, there is no PeerManagerActor in between